Зарплаты/Зарплаты/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Зарплаты
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var sum = 0;
+            Console.WriteLine("Введите число работников ");
+            int workers = int.Parse(Console.ReadLine());
+
+            for (var i=1; i<=workers; i++)
```

Comment:
Традиционно такой цикл пишут так
```suggestion
            for (var i = 0; i < workers; i++)
```